rotkehlchen/chain/scroll/modules/syncswap/decoder.py

```diff
@@ -0,0 +1,60 @@
+from typing import TYPE_CHECKING, Final
+
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.chain.evm.decoding.uniswap_v2_like.decoder import UniswapV2LikeDecoder
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import A_WETH_SCROLL
+from rotkehlchen.types import ChecksumEvmAddress
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.scroll.node_inquirer import ScrollInquirer
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+# https://syncswap.gitbook.io/api-documentation/resources/smart-contract#scroll-alpha-testnet
+SYNCSWAP_ROUTER_ADDRESS: Final = string_to_evm_address('0x80e38291e06339d10AAB483C65695D004dBD5C69')  # noqa: E501
+SYNCSWAP_VAULT_ADDRESS: Final = string_to_evm_address('0x7160570BB153Edd0Ea1775EC2b2Ac9b65F1aB61B')
+CPT_SYNCSWAP: Final = 'syncswap'
+SYNCSWAP_LABEL: Final = 'Syncswap'
```

Comment:
Any variable that's only used once should not exist. Please for this and all other files just put them as literal constants where they are used.

Also anythign that's supposed to be a constant should be in the module's `constants/py`